modules/ort/src/main/kotlin/resolver/OrtResultArtifactResolver.kt

```diff
@@ -97,6 +97,7 @@ class OrtResultArtifactResolver(result: OrtResult) : Function<Package, Artifact>
 
     private fun mapCopyrights(pkg: Package): CopyrightStatement? =
         licenseFindings[pkg.id]?.keys?.flatMap { it.copyrights }
+                ?.takeUnless { it.isEmpty() }
```

Comment:
Please rebase, this change (and the version updates below) should be in `master` now.

Reply:
done